src/core/localization/translations/en/common-messages.json

```diff
@@ -76,7 +76,9 @@
         "settled_area.tooltip": "Settled Area",
         "loss.tooltip": "Estimated loss"
       },
-      "no_event_in_feed": "Event not found in the current event feed"
+      "no_event_in_feed": "Event not found in the current event feed",
```

Comment:
I see that you branched out not from the latest main
You need it for lint-staged step to update .pot file with the new keys

Reply:
now pls just run` npm run i18n:export` 
it will update .pot file

Reply:
did it!